features/location/constants.ts

```diff
@@ -0,0 +1 @@
+export const EARTH_RADIUS_METERS = 6371e3;
```

Comment:
Is `features/` the best location for this? I was thinking that `features/` would house code that have a mix of JSX and other stuff.

For pure functions, I think `utils/` makes more sense?

So, `utils/geolocations/*` seems like a better place for this code.